src/epics/sync.js

```diff
@@ -56,15 +50,15 @@ export const getFetchQueriesEpic = fetchQueriesFn => (action$, store) =>
 			const { config, auth } = store.getState();
 			const fetch = fetchQueriesFn(config.apiUrl, { method: 'GET', auth });
 			return Observable.fromPromise(fetch(queries))  // call fetch
-				.takeUntil(action$.ofType(LOCATION_CHANGE, 'LOCATION_SYNC'))  // cancel this fetch when nav happens
+				.takeUntil(action$.ofType(LOCATION_CHANGED, 'LOCATION_SYNC'))  // cancel this fetch when nav happens
 				.map(apiSuccess)                             // dispatch apiSuccess with server response
 				.flatMap(action => Observable.of(action, apiComplete()))  // dispatch apiComplete after resolution
 				.catch(err => Observable.of(apiError(err)));  // ... or apiError
 		});
 
-export default function getSyncEpic(routes, fetchQueriesFn=fetchQueries) {
+export default function getSyncEpic(fetchQueriesFn=fetchQueries) {
 	return combineEpics(
-		getNavEpic(routes),
+		navEpic,
 		resetLocationEpic,
 		getFetchQueriesEpic(fetchQueriesFn)
```

Comment:
nitpick, but can you drop the `get` on here and call it `fetchQueriesEpic` for consistency?